oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/registration/ClientRegistrations.java

```diff
@@ -70,7 +78,7 @@
 	 * Example usage:
 	 * </p>
 	 * <pre>
-	 * ClientRegistration registration = ClientRegistrations.fromOidcIssuerLocation("https://example.com")
+	 * ClientRegistration registration = ClientRegistrations.withOidcIssuerLocation("https://example.com")
```

Comment:
Ups! Gonna change them back to `fromOidcIssuerLocation` as well as below `fromOAuth2IssuerLocation` in my final changes :+1: